docs/intro/installation.rst

```diff
@@ -32,6 +33,48 @@ exist, and they need to be installed separately, if needed.
 .. figure:: ../artwork/src/install.svg
    :width: 70%
 
+Python 3 (all operating systems)
+""""""""""""""""""""""""""""""""
+
+You should make sure that you have Python 3 installed on your system.
+The easiest way to do this is to open a terminal and type "python" to open a minimalistic Python session::
+
+   $ python
+   Python 3.9.1+ (default, Jan 20 2021, 14:49:22)
+   [GCC 10.2.1 20210110] on linux
+   Type "help", "copyright", "credits" or "license" for more information.
+   >>>
+
+If this fails, or reports a Python version with a leading ``2``, such as ``Python 2.7.18``, try typing ``python3`` (which some systems use to disambiguate between Python 2 and Python 3).
+If this fails, too, you need to obtain a recent release of Python 3.
+
+If you are on Windows, please note that you should **not** install Python via the Windows store, even if it opens after you typed ``python``, as this version requires additional configurations by hand (in particular of your ``$PATH`` :term:`environment variable`).
+Please instead check the Windows section at the end of the page for more convenient alternatives.
+
+.. findoutmore:: Python 2, Python 3, what's the difference?
+
+   Python 2 is an outdated, in technical terms "deprecated", version of Python.
+   Although it still exist as the default Python version on many systems, it is no longer maintained since 2020, and thus, most software has dropped support for Python 2.
+   If you only run Python 2 on your system, most Python software, including DataLad, will be incompatible, and hence unusable, resulting in errors during installation and execution.
+
+   But does that mean that you should uninstall Python 2?
+   **No**.
+   Keep it installed, especially if you are using Linux or MacOS.
+   Python 2 existed for 20 years and numerous software has been written in it.
+   It is quite likely that some basic operating system components or legacy software on your computer is depending on it, and uninstalling a preinstalled Python 2 from your system will likely render it unusable.
+   Install Python 3, and have both versions coexist peacefully.
```

Comment:
```suggestion
   Install Python 3, and have both versions coexist peacefully.
```

hahaha 

Did you mean to write:

> ... coexist peacefully (as long as you use some sort of virtual environment, otherwise things could get a lot less peaceful very quickly). 

:wink: 

Just kidding. This is perfect.